batchglm/train/numpy/glm_poisson/model_container.py

```diff
@@ -0,0 +1,120 @@
+from typing import Callable, Union
+
+import dask
+import numpy as np
+import scipy
+
+from .external import BaseModelContainer
+
+
+class NoScaleError(Exception):
```

Comment:
not so sure if we want to have this in `model_container.py`. Maybe in a separate file. But anyways, this proves that we don't need `init_theta_scale` I guess since it is handled here.

Reply:
In the other PR you mentioned that `ll` as a utility function should be in the `ModelContainer` file - I could make an `exceptions.py` just for this though.